### DISCUSSION TITLE
installed app update, ended up in maintenence mode??

### DISCUSSION THREAD

**Discussion by UltraBlackLinux** [nextcloud/all-in-one#4719] [Questions] [answered]
I just updated snappymail, and now nextcloud is in maintenance mode?
I just randomly had some issues where all of a sudden collabora was completely gone and then I restarted the container to be greeted by maintenance mode?
I thought the AIO image is supposed to be stable?
I don't even know what the nextcloud cointainer is called so I can disable maintenance mode manually... (This AIO container is driving me crazy. It's being more annoying that it should be)

Edit: Found the container via `docker ps` ran `occ maintenance:repair` and `occ maintenance:mode --off` and everything is working, but come on why did it do that?

**Comment by szaimen** [comment, marked as answer]
Hi, yes AIO ist stable. However the snappymail update had a bug and caused Nextcloud to fall into maintenance mode. So this need to be fixed on their end and nothing we can fix in AIO unfortunately.

**Comment by vawaver** [comment]
Greeting,
I am not IT pro.
I updated my apps today and my server failed to load - maintenance mode is active.
Can you please help me to fix it?
What exactly must I do?

```
Nextcloud is in maintenance mode, no apps are loaded.

Commands provided by apps are unavailable.

Nextcloud is in maintenance mode, no apps are loaded.

Commands provided by apps are unavailable.

+ '[' true = true ']'

+ '[' 11000 = 443 ']'

+ set +x

[12-Jun-2024 20:05:46] NOTICE: fpm is running, pid 557

[12-Jun-2024 20:05:46] NOTICE: ready to handle connections

Activating Collabora config...

Nextcloud is in maintenance mode, no apps are loaded.

Commands provided by apps are unavailable.

                                                                   

  There are no commands defined in the "richdocuments" namespace.  
  ```

**Reply by szaimen** [reply]
> Can you please help me to fix it?

You could restore a backup from the aio interface

**Comment by vawaver** [comment]
So I used these commands and Nextcloud is working again:
```
sudo docker exec --user www-data -it nextcloud-aio-nextcloud php occ maintenance:repair
```
and then
```
sudo docker exec --user www-data -it nextcloud-aio-nextcloud php occ maintenance:mode --off
```
